cacti/tests/conftest.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+from copy import deepcopy
+
+from datadog_checks.dev import docker_run
+import docker
+from datadog_checks.dev.subprocess import run_command
+from datadog_checks.cacti import CactiCheck
+
+from .common import (
+    HERE,
+    HOST,
+    INSTANCE_INTEGRATION
+)
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    env = {'HOST': HOST}
+    with docker_run(
+        compose_file=os.path.join(HERE, "compose", "docker-compose.yaml"),
+        env_vars=env,
+    ):
+        yield INSTANCE_INTEGRATION
+
+
+@pytest.fixture
+def check():
+    container = _get_container()
+    commands = ["apt-get update",
+                "apt-get install rrdtool librrd-dev libpython-dev build-essential -y",
+                "pip install rrdtool"]
+    for command in commands:
+        exit_code, output = container.exec_run(command)
```

Comment:
Another concern here, the `check` method will be run once for every integration test. I agree that for now we only have one test but still, those commands should only be run once for the whole session and should be moved to a place where they run during initialization only.

Reply:
I'd like them to be part of the `ddev env start` and have opened a feature request for that

Reply:
I think that's done with `start_commands` no? See the ibm_db2 setup.

Reply:
IBM_DB2 seems is setting up the service machine, while I'm setting up the agent machine. If I try to emulate IBM_DB2 it cannot find the docker container for the agent, so I guess it starts the service one first?